contrib/terraform/aws/README.md

```diff
@@ -99,6 +108,31 @@ data "aws_ami" "distro" {
 
   owners = ["688023202711"]
 }
+```
+
+## Connecting to Kubernetes
+
+You can use the following set of commands to get the kubeconfig file from your newly created cluster. Before running the commands, make sure you are in the project's root folder.
```

Comment:
Kubespray has an option `kubeconfig_localhost` for getting kubeconfig from remote deployed cluster.

We just need to set the option to `true` instead of the default `false` and run `cluster.yml`.
Then we can get the kubeconfig from `artifacts` directory under our inventory like `./inventory/sample/artifacts/admin.conf`.

Doesn't the option work for your use case?

Reply:
Thanks for your response @medined 

> No, setting kubeconfig_localhost to true does not help in my situation.
> The `true` settings does create the local file `./inventory/artifacts/admin.conf`
> but the IP address of the cluster is still the private IP instead of the load balancer.

I see.
One more question is how about putting bastion machine on the same private network as target machines on AWS?
I am doing the same thing on Azure and `kubectl` works on the same private network with kubeconfig which contain private IP address.

I felt it could not be most general way to modify kubeconfig to public IP address if the above access is possible on AWS.